packages/block-library/src/latest-posts/edit.js

```diff
@@ -238,7 +238,7 @@ function Controls( { attributes, setAttributes, postCount } ) {
 						isShownByDefault
 					>
 						<RadioControl
-							label={ __( 'Show' ) }
+							label={ __( 'Content Length' ) }
```

Comment:
```suggestion
							label={ __( 'Content length' ) }
```

We use sentence case in feature labels. See [the Copy Guidelines](https://github.com/WordPress/gutenberg/blob/trunk/docs/contributors/documentation/copy-guide.md#five-pay-attention-to-capitalization).